src/featurestorebundle/delta/target/reader/DeltaTableTargetsReader.py

```diff
@@ -16,9 +16,14 @@ def __init__(
         self.__spark = spark
         self.__table_names = table_names
 
-    def read(self, entity: str) -> DataFrame:
-        full_table_name = self.__table_names.get_entity_targets_full_table_name(entity)
+    def read(self, entity_name: str) -> DataFrame:
+        full_table_name = self.__table_names.get_targets_full_table_name(entity_name)
 
-        self.__logger.info(f"Reading targets for entity {entity} from table {full_table_name}")
+        self.__logger.info(f"Reading targets for entity {entity_name} from table {full_table_name}")
+
+        return self.__spark.read.table(full_table_name)
```

Comment:
so target_store has to be a table? not a delta file?

Reply:
Can be delta file, it has configurable backend